data/json/items/gun/shot.json

```
@@ -102,8 +102,8 @@
     "copy-from": "shotgun_pump",
     "looks_like": "remington_870",
     "type": "GUN",
-    "name": { "str": "14-round tube shotgun" },
-    "description": "This shotgun uses a system of 2 tube magazines to carry 14 rounds of 12 gauge shot.  Each tube has to be loaded separately, but this offers the option of loading different ammunition for different situations.",
+    "name": { "str": "12 gauge Kel-Tec bullpup shotgun" },
```

I'd drop the brand name, it doesn't add anything.
```suggestion
    "name": { "str": "12 gauge bullpup shotgun" },
```